modules/serverless-install-web-console.adoc

```diff
@@ -25,8 +25,8 @@ image::serverless-create-sub.png[Create Operator Subscription page]
 .. The *Installation Mode* is *All namespaces on the cluster (default)*. This mode installs the Operator in the default `openshift-operators` namespace to watch and be made available to all namespaces in the cluster.
 .. The *Installed Namespace* will be `openshift-operators`.
 .. Select an *Update Channel*.
-... The *4.3* channel will enable installation of the latest stable release of the {ServerlessOperatorName}.
-... The *preview-4.3* channel enables installation of the latest preview version of the {ServerlessOperatorName}, which may contain features that are not yet available from the *4.3* update channel.
+... The *4.4* channel will enable installation of the latest stable release of the {ServerlessOperatorName}.
+... The *preview-4.4* channel enables installation of the latest preview version of the {ServerlessOperatorName}, which may contain features that are not yet available from the *4.4* update channel.
```

Comment:
Please revert this - for now it's just `preview-4.3`, and for OCP 4.4 I have a separate PR in to update this to just the `4.4` channel as there is no preview channel yet @ahardin-rh

Reply:
Thank you for confirming!